### PR TITLE
add JUG-PE as adopter

### PR DESCRIPTION
**Pull request by vinyfalcao**
Signed-off-by: Vinicius Falcão <vinyfalcao@hotmail.com>

### CODE REVIEW

File: data/spec_adopters.yml [hunk 1]
```diff
@@ -13,6 +13,10 @@ adopters:
       homepage_url: https://www.meetup.com/MadrasJUG/
       logo: mjug.svg
       logo_white: mjug.svg
+    - name: JUG-PE
+      homepage_url: https://jugpe.com.br/
+      logo: jugpe.svg
```
Comment (autumnfound) [inline data/spec_adopters.yml]: This looks like it should be jugpe.png, not svg

Comment (vinyfalcao) [reply data/spec_adopters.yml]: Sorry about that, fixed

**Comment by chrisguindon** [comment]
A preview is available here: 
https://deploy-preview-690--jakartaee.netlify.com/specifications/jsonp/

**Comment by TanjaObradovic** [comment]
Any chance to reduce the size of all logos to half a the size?

**Pull request merged**